### ISSUE TITLE
‘Help’ icon button is not accessible while navigating with keyboard in scan mode, present on ‘Pages’ popup.

### ISSUE THREAD

**Issue by azaslonov** [paperbits/paperbits-demo#99]
User Experience:

User rely on keyboard/Screen reader will find difficulty to interact with the controls if controls are not accessible through keyboard as user won't be able to perform functionality over such control.

 

Note: User credentials should NOT be included in the bug.

 

Repro Steps:

1. Open Azure portal with valid credentials and create a API Management.
2. Click Developer Portal button to navigate to the portal.
3. Wait portal to load in design mode.
4. Navigate to the Left nav and select Pages button.
5. Navigate to the various controls of the "Pages" window.

6. Verify the issue.
 

Actual Result:

‘Help’ icon button is not accessible while navigating with keyboard in scan mode, present on ‘Pages’ popup.

 
Expected Result:

‘Help’ icon button should be accessible while navigating with keyboard in scan mode, present on ‘Pages’ popup.

Note:

Similar issue exist with Chrome.
Similar issue exist throughout the application for same controls.

MAS Reference:

MAS 4.2.9 - Execution of Available Actions (33I)

Reference Links

Accessibility Insights! - Identify accessibility bugs before check-in and make bug fixing faster and easier.
External Bug Process: If this bug belongs to external team, mark it as resolved/done and assign it back to the tester with notes on where to file/route the bug. For more information please use this Link. “HCL Staff should not log any third party external bugs. Should be routed to edad team”.
Please reach out to C&AI Teams channel for any process related queries.

**Comment by azaslonov** [comment]
Fixed.
